modules/host/networking.nix

```diff
@@ -6,6 +6,12 @@
     firewall.allowedUDPPorts = [67]; # DHCP
     useNetworkd = true;
   };
+
+  networking.nat = {
```

Comment:
Address host NAT in the minimal host documentation

Reply:
This enables NAT from host to the VMs, i.e. you can share the host's network with VMs. This is not needed at all other than debugging purposes

Reply:
Ok. This should go only via nix development-modules then. Even if I'd like to see "no networking on host" in development mode as well, it may not be very practical at this phase in development when tools from nix caches are often quite helpful to try/debug something in the development. Making the host no-networking/read-only (like netvm) would require reflashing for any small change.